### DISCUSSION TITLE
XP Pen Innovator 16 - Settings are not being applied

### DISCUSSION THREAD

**Discussion by LangeRII** [OpenTabletDriver/OpenTabletDriver#3648] [Q&A]
Hello guys! The problem is as simple as it is frustrating. No matter what setting I change, it just doesn't get applied. It starts with something as simple as switching from absolute to relative mode: I select the relative mode, press "apply" and "save," but it stays in absolute mode.

I followed this tutorial step by step:
https://www.youtube.com/watch?v=7PtZVdkyZhs

I also tried for half the night with jamesbt365 on Discord, but unfortunately, without success. Still, thanks at this point!

I completely uninstalled the old drivers, cleaned it up with tabletdrivercleanup, reinstalled everything from scratch, but it just won't apply the settings. The tablet is recognized, it works, but the settings just won't be applied. Can someone please, please help me? Thanks so much!

Here are the diagnostics: [Pen1.json](https://github.com/user-attachments/files/18268642/Pen1.json)

**Comment by gonX** [comment]
Does this still happen with 0.6.5.1? I'd also double check to make sure the OEM drivers aren't installed. The diag doesn't indicate anything abnormal at a brief glance.
